CMakeLists.txt

```diff
@@ -650,6 +650,23 @@ set(
 )
 mark_as_advanced(HELICS_CMAKECONFIG_INSTALL_DIR)
 
+set(HELICS_INSTALL_TARGETS helics_base helics_base_includes jsoncpp_lib helics-static)
+if(BUILD_SHARED_LIBS)
+    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helics-shared)
+endif()
+if(BUILD_C_SHARED_LIB)
+    set(HELICS_INSTALL_TARGETS ${HELICS_INSTALL_TARGETS} helicsSharedLib helicsCpp98)
+endif()
+if(BUILD_APPS)
```

Comment:
Should some like this be in the respective build directories instead in the main level cmake?

Reply:
Possibly, I wasn't sure about the effect of changing scopes would have on setting the variable. I think there's a much simpler way of doing this that I should use instead.